third_party/move/move-prover/boogie-backend/src/options.rs

```diff
@@ -15,7 +15,6 @@ const DEFAULT_BOOGIE_FLAGS: &[&str] = &[
     "-printVerifiedProceduresCount:0",
     "-printModel:1",
     "-enhancedErrorMessages:1",
-    "-monomorphize",
```

Comment:
could you clarify more on the reason to remove this option?

Reply:
First of all, this flag `-monomorphize` does not exist any more, but seem to be merged into the `typeEncoding` option. And, `monomorphic` is by default.
```
/typeEncoding:<t>
                Encoding of types when generating VC of a polymorphic program:
                   m = monomorphic (default)
                   p = predicates
                   a = arguments
                Boogie automatically detects monomorphic programs and enables
                monomorphic VC generation, thereby overriding the above option.
                If the latter two options are used, then arrays are handled via axioms.
```

Reply:
The reference for this change in Boogie: https://github.com/boogie-org/boogie/pull/732